portfolio/src/main/webapp/script.js

```diff
@@ -131,7 +131,7 @@ function uploadComment() {
     }
   }).then(function(response) {
     if (!response.ok) {
-      // TODO: handle case when verification fails
+      console.error();
```

Comment:
This func behave similar to `log`, you can add a hardcoded error cause or read from the response.